scripting/futuristicgrenades.sp

```diff
@@ -283,6 +285,34 @@ public int Native_GetAmountForcefields(Handle plugin, int numParams)
 	return g_hForcefields.Length;
 }
 
+public int Native_ChangeMode(Handle plugin, int numParams)
+{
+	int client = GetNativeCell(1);
+	
+	if (client < 1 || client > MaxClients)
+	{
+		return ThrowNativeError(SP_ERROR_NATIVE, "Invalid client index (%d)", client);
+	}
+	if (!IsClientConnected(client))
+	{
+		return ThrowNativeError(SP_ERROR_NATIVE, "Client %d is not connected", client);
+	}
+	
+	int iDecoyMode = GetNativeCell(2);
+	int iForceFieldMode = GetNativeCell(3);
+	int iExplosionMode = GetNativeCell(4);
+	int iImplosionMode = GetNativeCell(5);
+	
+	g_eMode[client] = view_as<DecoyMode>(iDecoyMode);
+	g_eForcefieldMode[client] = view_as<ForceFieldMode>(iForceFieldMode);
+	g_eForceExplosionMode[client] = view_as<ForceExplosionMode>(iExplosionMode);
+	g_eForceImplosionMode[client] = view_as<ForceImplosionMode>(iImplosionMode);
+	
```

Comment:
And maybe like `OnPlayerRunCmd` add PrintActiveSettings and EmitSoundToClientAny if player has the decoy in the hand